src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayTableViewCell.swift

```diff
@@ -377,3 +395,12 @@ class DiaryOverviewDayTableViewCell: UITableViewCell {
 		didTapClickableView?()
 	}
 }
+
+extension UIStackView {
```

Comment:
I thought that's what src/xcode/ENA/ENA/Source/Extensions/UIStackView+Background.swift is for?

Reply:
That was the test code before I moved into an extension and forgot to remove it.